esmvalcore/cmor/tables/custom/CMOR_sfcWind.dat

```diff
@@ -0,0 +1,23 @@
+SOURCE: CMIP5
```

Comment:
Is the custom CMOR table entry really needed? If I remember correctly, the definition of derived variables will automatically be taken from another mip table (from the same project) if they are not available in the mip table that is specified in the recipe.

Reply:
no, indeed it's not needed - but it shows up in pretty much all other custom tables, so we shouldn't fix this here, but in a dedicated PR

Reply:
So the problem was, that when I ran the derivation, it wasn't working without the custom table see [here](https://github.com/ESMValGroup/ESMValCore/issues/1895#issuecomment-1741596407l). Based on the error message, I attributed it to the fact that `uas` and `vas` are in `3hr` table, while `sfcWind` is in `CF3hr` or `E3hr`, but I'm not sure that is the reason now.

Reply:
let's tackle that in a separate PR - good points from both Bouwe and Liza, but it's just not the place for it here

Reply:
It seems to work without the custom CMOR table for me. I tried this recipe:
```yaml
documentation:
  description: |
    Example recipe that plots a map and timeseries of temperature.

  title: Recipe that runs an example diagnostic written in Python.

  authors:
    - andela_bouwe
    - righi_mattia

  maintainer:
    - schlund_manuel

  references:
    - acknow_project

  projects:
    - esmval
    - c3s-magic

datasets:
  - {dataset: ERA5, project: native6, tier: 3, type: reanaly}

diagnostics:

  map:
    description: Global map of temperature in January 2000.
    themes:
      - phys
    realms:
      - atmos
    variables:
      sfcWind:
        mip: E1hr
        timerange: 1990/PT1M
        derive: true
        caption: |
          Global map of {long_name} in January 2000 according to {dataset}.
    scripts:
      script1:
        script: examples/diagnostic.py
        quickplot:
          plot_type: pcolormesh
          cmap: Reds
```
@malininae Could you try again and share the error message?

Reply:
You're right, the 'derived' argument is only used for looking things up in the custom CMOR table. I've opened #2256 to improve that.